### PR TITLE
Improvements/937

### PR DESCRIPTION
**Pull request by RomanGalochkin**
Please review the previous PR.

### CODE REVIEW

File: packages/client/src/components/measurables/MeasurableForm.re [hunk 1]
```diff
@@ -557,7 +557,7 @@ module Edit = {
                   },
                   (),
                 )##variables,
-              ~refetchQueries=[|"getMeasurables"|],
+              ~refetchQueries=[|"getMeasurables", "getMeasurable"|],
```
Comment (OAGr) [inline packages/client/src/components/measurables/MeasurableForm.re]: good catch

File: packages/server/migrations/js/201912230755000-notebooks-index.js [hunk 2]
```diff
@@ -32,6 +35,10 @@ module.exports = {
         'Notebooks_name_ownerId_unique',
       );
 
+      // @todo: Here first of all we should have to remove
+      // @todo: duplicates and only then to create another
+      // @todo: unique index.
```
Comment (OAGr) [inline packages/server/migrations/js/201912230755000-notebooks-index.js]: When do you expect that we do these @todos? Can you make a new story or two and prioritize accordingly?

File: packages/server/src/graphql/types/measurables.js [hunk 3]
```diff
@@ -7,15 +7,37 @@ const resolvers = require('../resolvers');
...
+const name = {
+  type: graphql.GraphQLNonNull(scalars.string3to512),
+};
+const labelSubject = {
+  type: graphql.GraphQLString,
+  description: 'Description of a question\'s entity',
+};
+const labelOnDate = {
+  type: DateType.default,
+  description: 'Description of a question\'s date',
```
Comment (OAGr) [inline packages/server/src/graphql/types/measurables.js]: Kudos for adding these descriptions!

File: packages/server/migrations/js/201912230755000-notebooks-index.js (hunk 2 above)
Comment (RomanGalochkin) [reply packages/server/migrations/js/201912230755000-notebooks-index.js]: It is a todo not in a direct sense. It is like a remark. I am not sure that we should do something here this is why I did not create a new issue.

Comment (OAGr) [reply packages/server/migrations/js/201912230755000-notebooks-index.js]: Ok, that works

**Pull request merged**